consensus/istanbul/proxy/proxy_engine.go

```diff
@@ -66,11 +67,10 @@ type proxyEngine struct {
 	logger  log.Logger
 	backend BackendForProxyEngine
 
-	// Proxy's validator
-	// Right now, we assume that there is at most one proxied peer for a proxy
-	// Proxy's validator
-	proxiedValidator   consensus.Peer
-	proxiedValidatorMu sync.RWMutex
+	// Proxied Validators set and count of nodekey addresses
```

Comment:
comment fix: "count of nodekey addresses"

Reply:
Nice catch